libs/common/src/constants/errors/error.messages.ts

```diff
@@ -1,8 +1,19 @@
-export const ERROR_MESSAGES = {
-  INVALID_CREDENTIALS: 'Invalid credentials',
-  USER_NOT_FOUND: 'User not found',
-  USER_ALREADY_EXISTS: 'User already exists',
-  USERNAME_TAKEN: 'Username already taken',
-  CREATE_USER: 'Error creating user',
-  LOGOUT_FAILED: 'Error logging out',
-};
+export const ERROR_MESSAGES = {
+  CREATE_TRANSCRIPT_ERROR: 'Error creating transcript',
+  UPDATE_TRANSCRIPT_ERROR: 'Error updating transcript',
+  GET_TRANSCRIPTS_ERROR: 'Error fetching transcripts',
+  GET_FILE_ERROR: 'Error fetching file',
+  DELETE_TRANSCRIPT_ERROR: 'Error deleting transcript',
+  CREATE_CHAT_ERROR: 'Error creating chat',
+  UPDATE_CHAT_ERROR: 'Error updating chat',
+  GET_CHATS_ERROR: 'Error fetching chats',
+  DELETE_CHAT_ERROR: 'Error deleting chat',
+  GET_CHAT_ERROR: 'Error fetching chat',
+  INVALID_CREDENTIALS: 'Invalid credentials',
+  USER_NOT_FOUND: 'User not found',
+  USER_ALREADY_EXISTS: 'User already exists',
+  USERNAME_TAKEN: 'Username already taken',
+  CREATE_USER: 'Error creating user',
+  LOGOUT_FAILED: 'Error logging out',
+  CHAT_NOT_FOUND: 'Chat not found',
+};

```

Comment:
Could we separate these into different for files? For example, chat.error.message.ts , authentication.messages.ts